airbyte-integrations/connectors/destination-bigquery-denormalized/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDenormalizedDestination.java

```diff
@@ -46,10 +61,19 @@ protected boolean isDefaultAirbyteTmpTableSchema() {
   }
 
   public static void main(final String[] args) throws Exception {
+    initSentry();
     final Destination destination = new BigQueryDenormalizedDestination();
-    LOGGER.info("starting destination: {}", BigQueryDenormalizedDestination.class);
-    new IntegrationRunner(destination).run(args);
-    LOGGER.info("completed destination: {}", BigQueryDenormalizedDestination.class);
+    ITransaction transaction = Sentry.startTransaction("IntegrationRunner()", "run");
+    try {
+      LOGGER.info("starting destination: {}", BigQueryDestination.class);
+      new IntegrationRunner(destination).run(args);
+    } catch (Exception e) {
+      transaction.setThrowable(e);
+      transaction.setStatus(SpanStatus.INTERNAL_ERROR);
+    } finally {
+      transaction.finish();
+      LOGGER.info("completed destination: {}", BigQueryDestination.class);
+    }
   }
```

Comment:
The Sentry transaction can probably be moved to the integration runner itself. But I think this can be done later.